func_adl_xAOD/common/cpp_ast.py

```diff
@@ -11,7 +11,7 @@
                                                      cpp_variable)
 from func_adl_xAOD.common.cpp_vars import unique_name
 from func_adl_xAOD.common.util_scope import gc_scope
-
+import func_adl_xAOD.common.cpp_representation as crep
```

Comment:
You are importing crep to get at `cpp_value` here. Not above that `cpp_representation` is already imported. Perhaps just add `cpp_value` to the list there and then use it below as `cpp_value` rather than `crep.cpp_value`? That will make it consistent. If you like you can also delete the above and replace all the references with `crep.xxx`, but I'd suggest the first.